public/javascripts/SVLabel/css/svl.css

```diff
@@ -8,10 +8,10 @@
 text {
     visibility: hidden;
 }
-
+/*
```

Comment:
But all of these elements (sound and stats) are within the container element right? Why is it showing up?

There is no reason for those elements to show up when others aren't. Maybe change the original html code for these elements so that it conforms with the visibility of the other elements?